README.md

```diff
@@ -1,2 +1,119 @@
 # wavefront-lambda-go
-Wavefront Go Wrapper for AWS Lambda
+
+This is a Wavefront Go wrapper for AWS Lambda to enable reporting standard lambda metrics and custom app metrics directly to wavefront.
+
+## Requirements
+Go 1.x
+
+## Installation
+```
+go get github.com/wavefrontHQ/wavefront-lambda-go.git
```

Comment:
Add a link to install go from https://golang.org/doc/install as the first step and then the `go get ...`